Wikipedia.xcodeproj/project.pbxproj

```diff
@@ -3399,7 +3399,7 @@
 		676070A1227CE60800A81F09 /* TalkPageReplyFooterView.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = TalkPageReplyFooterView.swift; sourceTree = "<group>"; };
 		676070A32280987C00A81F09 /* TalkPageTopicNewViewController.xib */ = {isa = PBXFileReference; lastKnownFileType = file.xib; path = TalkPageTopicNewViewController.xib; sourceTree = "<group>"; };
 		676A8A8223A4013D0084B967 /* ArticleFetcher.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; name = ArticleFetcher.swift; path = ../Wikipedia/Code/ArticleFetcher.swift; sourceTree = "<group>"; };
-		6771298E24FF76AC00E89CA5 /* ArticleAsLivingDocViewController.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = ArticleAsLivingDocViewController.swift; sourceTree = "<group>"; };
+		6771298E24FF76AC00E89CA5 /* ArticleAsLivingDocViewController.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = ArticleAsLivingDocViewController.swift; sourceTree = "<group>"; usesTabs = 0; };
```

Comment:
Not a bug: I don't know what this flag does, and a quick google doesn't pop anything up. But many other files have the same flag - possibly something from the new Xcode version and how it handles window/tab management? Happy to approve the PR.

Reply:
This one's actually about using tabs vs. spaces. In the File inspector in Xcode when any file is selected, the bottom Text Settings section has the option to explicitly select which indent type the file should use (regardless of Xcode's settings). Since the files in the project mostly all use spaces for indentation, I just try to explicitly set it to "Spaces" when I'm working on files I haven't touched before.

Reply:
Awesome, thanks!